<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderImbalanceIndicatorWindow</class>
 <widget class="QWidget" name="OrderImbalanceIndicatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imbalance Indicator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Resources.qrc">
    <normaloff>:/icons/scales.png</normaloff>:/icons/scales.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QTableView" name="m_orderImbalanceIndicatorTableView">
     <property name="autoScroll">
      <bool>false</bool>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="m_parametersLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="Spire::UI::CollapsibleWidget" name="m_timeRangeParametersWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Spire::UI::CollapsibleWidget" name="m_marketSelectionWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spire::UI::CollapsibleWidget</class>
   <extends>QWidget</extends>
   <header>Spire/UI/CollapsibleWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
